networking/network_observability/network-observability-overview.adoc

```diff
@@ -32,23 +32,28 @@ The Network Observability Operator provides the Flow Collector API custom resour
 {product-title} console integration offers overview, topology view and traffic flow tables.
 
 [id="network-observability-dashboards"]
-=== Network Observability metrics
+=== Network Observability metrics dashboards
 
-The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
+The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics. Infrastructure and Application metrics are shown in a split-view for namespace, pod, and workloads. 
 
 In *Observe* -> *Dashboards*, the *Netobserv* dashboard provides a quick overview of the network flows in your {product-title} cluster. You can view distillations of the network traffic metrics in the following categories:
 
- * *Top flow rates per source and destination namespaces (1-min rates)*
- * *Top byte rates emitted per source and destination nodes (1-min rates)*
- * *Top byte rates received per source and destination nodes (1-min rates)*
- * *Top byte rates emitted per source and destination workloads (1-min rates)*
- * *Top byte rates received per source and destination workloads (1-min rates)*
- * *Top packet rates emitted per source and destination workloads (1-min rates)*
- * *Top packet rates received per source and destination workloads (1-min rates)*
+ * *Top byte rates received per source and destination nodes*
+ * *Top byte rates received per source and destination namespaces*
+ * *Top byte rates received per source and destination workloads*
 
 You can configure the `FlowCollector` `spec.processor.metrics` to add or remove metrics by changing the `ignoreTags` list. For more information about available tags, see the xref:../../networking/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference]
 
-Also in *Observe* -> *Dashboards*, the *Netobserv/Health* dashboard provides metrics about the health of the Operator.
+Also in *Observe* -> *Dashboards*, the *Netobserv/Health* dashboard provides metrics about the health of the Operator in the following categories.
+
+* *Flows*
+* *Flows Overhead*
+* *Top flow rates per source and destination nodes (1-min rates)*
+* *Top flow rates per source and destination namespaces (1-min rates)*
+* *Top flow rates per source and destination workloads (1-min rates)*
```

Comment:
```suggestion
* *Top flow rates per source and destination nodes*
* *Top flow rates per source and destination namespaces*
* *Top flow rates per source and destination workloads*
```

Either we keep the `(1-min rates)` mentions everywhere or nowhere.

Reply:
I'll remove

Reply:
Addressed!